yaks-html/lib/yaks/format/html.rb

```diff
@@ -39,8 +39,8 @@ def render_resource(resource, templ = section('resource'))
           .replace('.forms') {|div| render_forms(resource.forms).call(div) }
           .replace('.subresource') {|sub_templ| render_subresources(resource, templ, sub_templ) }
       end
-      alias render_collection_resource render_resource
-      alias render_null_resource render_resource
+      alias_method :render_collection_resource, :render_resource
+      alias_method :render_null_resource, :render_resource
```

Comment:
not sure I like this one. Is that really what people prefer?

Seems there's [a reason](https://github.com/bbatsov/ruby-style-guide#alias-method), but it doesn't really explain the problem. You have a source for this?